browser/net/brave_static_redirect_network_delegate_helper.cc

```diff
@@ -15,6 +17,24 @@
 
 namespace brave {
 
+const char kSafeBrowsingTestingEndpoint[] = "foobar.com";
+
+namespace {
+
+bool safebrowsing_api_endpoint_for_testing_ = false;
```

Comment:
I think it should be `g_safebrowsing_api_endpoint_for_testing_`

Reply:
shouldn't it also be a string instead of a bool so we can pass the value in?

Reply:
1. Global strings are not allowed by the chromium linter.
2. const char arrays can cause overflow issues.